x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/configuration_step/configuration_step_form.tsx

```diff
@@ -445,6 +456,37 @@ export const ConfigurationStepForm: FC<CreateAnalyticsStepProps> = ({
       >
         <Fragment />
       </EuiFormRow>
+      {Array.isArray(unsupportedRuntimeFields) && unsupportedRuntimeFields.length > 0 && (
+        <>
+          <EuiCallOut size="s" color="warning">
+            <FormattedMessage
+              id="xpack.ml.dataframe.analytics.create.unsupportedRuntimeFieldsCallout"
+              defaultMessage="The runtime {runtimeFieldsCount, plural, one {field} other {fields}} {unsupportedRuntimeFields} {extraCountMsg} are not supported for analysis."
```

Comment:
If I'm not mistaken, in case `{extraCountMsg}` is empty, the sentence ends up with 2 spaces before `are not supported for analysis` string. I think `and {count} more` message should take care of adding a space.

Reply:
I think the extra space to `extraCountMsg` will actually be stripped out. For example, this is what I looks like with the spaces in `extraCountMsg`.
<img width="1168" alt="Screen Shot 2021-02-16 at 10 52 29" src="https://user-images.githubusercontent.com/43350163/108095296-52d9ec80-7045-11eb-8c3d-200574131d38.png">

And this is what it looks like as is:
<img width="1168" alt="Screen Shot 2021-02-16 at 10 32 30" src="https://user-images.githubusercontent.com/43350163/108095392-6b4a0700-7045-11eb-89a4-2dd46756561a.png">